mod-orders-storage/schemas/title.json

```diff
@@ -26,6 +26,42 @@
       "items": {
         "$ref": "product_identifier.json"
       }
+    },
+    "contributors": {
+      "description": "List of contributors to the material",
+      "id": "contributors",
+      "type": "array",
+      "items": {
+       "type": "object",
+        "$ref": "contributor.json"
+      }
+    },
+    "publisher": {
+      "description": "Publisher of the material",
+      "type": "string"
+    },
+    "edition": {
+      "description": "Edition of the material",
+      "type": "string"
+    },
+    "publishedDate": {
+      "description": "Edition of the material",
```

Comment:
description needs to be changes, and type must be date if it is publishedDate?

Reply:
oh I see the type needn't changed, we just expect it to be a year. just the description then

Reply:
Also the field is named "publicationDate" in the story, might want to correct that as well

Reply:
@craigmcnally , could you please clarify:
according to story description this field names "Publication Date", but according wiki page - "publishedDate". Is this OK to left it as is, i.e. "publishedDate"?

Reply:
Either way is fine.  If there's a strong preference to how this is displayed in the UI, we can use whatever label desired in the UI.   As long as the description is clear in the schema, the name of the field is of little consequence.